baseapp-message-templates/baseapp_message_templates/models.py

```diff
@@ -18,21 +21,16 @@ def db_type(self, connection):
         return "citext"
 
 
-class EmailTemplate(TimeStampedModel):
-    name = CaseInsensitiveCharField(
-        max_length=255,
-        unique=True,
-        help_text="Unique name used to identify this message",
-    )
-
-    # SendGrid
-    sendgrid_template_id = models.CharField(
-        max_length=255,
+class TemplateTranslation(TimeStampedModel):
+    target_content_type = models.ForeignKey(
+        ContentType,
         blank=True,
         null=True,
+        on_delete=models.CASCADE,
     )
-
-    # Core
+    target_object_id = models.PositiveIntegerField(blank=True, null=True)
+    target = GenericForeignKey("target_content_type", "target_object_id")
```

Comment:
Why are we using generic foreign keys here?  Are these going to provide translations for something other than EmailTemplate?